main.rb

```diff
@@ -0,0 +1,148 @@
+      print 'Age: '
+      age = gets.chomp.to_i
+      print 'Name: '
+      name = gets.chomp
+      if input == 1
+        print 'Has parents permission? [Y/N]: '
+        parent_permission = gets.chomp
+        Student.add_a_student(age, name, parent_permission)
+        puts "The student #{name}, Age: #{age} has been created."
+      elsif input == 2
+        print 'Specialization: '
+        specialization = gets.chomp
+        Teacher.add_a_teacher(age, name, specialization)
+        puts "The teacher #{name}, Age: #{age}, specialized in #{specialization} has been created."
+      else
+        puts 'Invalid option'
+      end
+      puts ''
+    when 4
+      # Create a book
+      print 'Enter the book title: '
+      title = gets.chomp.to_s
+      print 'Enter the book author: '
+      author = gets.chomp.to_s
+      Book.add_a_book(title, author)
+      puts "Your book #{title} by #{author} has been created and added to the library"
+      puts ''
+    when 5
+      # Create a rental
+      if books_count.positive?
+        if students_count.positive? || teachers_count.positive?
+          puts 'Select a book by its number from the following list: '
+          Book.list_all_books
+
+          selected_id = gets.chomp.to_i
+          book = Book.select_a_book(selected_id)
+
+          puts 'Select a person by its number from the following list: '
+          Person.list_all_people
+
+          selected_index = gets.chomp.to_i
+          person = Person.select_a_person(selected_index)
+          puts ''
+
+          if person
+            date = Date.today
+            puts "Date: #{date}"
+            Rental.add_a_rental(date, book, person)
+            puts 'Rental created successfully'
+          else
+            puts 'Person not found'
+            puts ''
+          end
+        else
+          puts 'Please Create a person'
+          puts ''
+        end
+      else
+        puts 'Books missing to create a Rental. Please create at least 1 Book and 1 person to process the operation.'
+        puts ''
+      end
+    when 6
+      # List all rentals for a given person id
+      if students_count.positive? || teachers_count.positive?
+        puts 'Enter the person id: '
+        Person.list_all_people
+
+        selected_id = gets.chomp.to_i
+        person = Person.select_a_person(selected_id)
+
+        person_rentals_count = person.rentals.count
+        puts "#{person.name} rented #{person_rentals_count} times."
+        if person_rentals_count.positive?
+          puts 'The following books were concerned:'
+          person.rentals.each do |rental|
+            puts "[#{rental.book.id}] #{rental.book.title} by #{rental.book.author}"
+            puts ''
+          end
+        end
+      else
+        puts 'No person registered in the database.'
+        puts ''
+      end
+    when 7
+      break
+    else
+      puts 'Invalid choice'
+      puts ''
+    end
+  end
+end
+
+# rubocop:enable Metrics
```

Comment:
- [ ] You did a great job shifting your code from `app.rb` to `main.rb`. I know it wasn't easy to move your code and make the adjustments to meet the requirements and make your application run successfully like it was before. But as per the requirement, you should not disable `rubocop` in the project's code snippet. Please remove the comment and resolve the linter issue before submitting for the next review.